src/Oceananigans.jl

```diff
@@ -142,18 +141,31 @@ import Base:
     push!
 
 "Boolean denoting whether AMGX.jl can be loaded on machine."
-const hasamgx = @static Sys.islinux() ? true : false
+# TODO: find a way to check whether the libraries for AMGX and NETCDF 
+# (libamgxsh and libnetcdf, respectively) are installed on the machine
+const hasamgx   = false #@static Sys.islinux() ? true : false

```

Comment:
why this change?

Reply:
We are using a system that is linux but does not have AMGX.